ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -36,19 +36,19 @@ vacuum_cost_delay = 20ms
 
 # WRITE AHEAD LOG
 
-{% if hot_standby_server|default(None) %}
 archive_mode = on
-archive_command = 'rsync -a %p postgres@{{ hot_standby_server }}:{{ archive_dir }}/%f'   # archive directory lives on first standby machine specified in the inventory file
+archive_command = 'test ! -f {{ postgresql_archive_dir }}/%f && cp %p {{ postgresql_archive_dir }}/%f'
```

Comment:
This changes the archiving to keep them locally instead of shipping them to the standby. PG will take care of removing them as needed according to any replication slots that are in use.